dmscripts/helpers/supplier_data_helpers.py

```diff
@@ -198,7 +198,11 @@ def remove_declaration(self, supplier_id, framework_slug):
                 f"Declaration of supplier {supplier_id} "
                 f"that applied to framework {framework_slug} removed"
             )
-            return self.api_client.remove_supplier_declaration(supplier_id, framework_slug, 'user')
+            return self.api_client.remove_supplier_declaration(
+                supplier_id,
+                framework_slug,
+                'Data Retention Script {}'.format(datetime.now().isoformat())
```

Comment:
This assumes that the only script to ever use this method will want to sign their audit event as `Data Retention Script *`.
It removes the generic element of this class and arguably its reason to belong in helpers.

Passing the actor/ user through from the script would open this up to be more generic again.

Reply:
@benvand I think this is fixed now